enterprise/api/v1/views.py

```diff
@@ -151,6 +151,7 @@ def contains_content_items(self, request, pk, course_run_ids, program_uuids):
 
         contains_content_items = False
         for catalog in enterprise_customer.enterprise_customer_catalogs.all():
+            # Note that these checks are not transitioned over
             contains_course_runs = not course_run_ids or catalog.contains_courses(course_run_ids)
```

Comment:
These calls are not transitioned over so that `edx-enterprise.contains_content_items` is not calling `enterprise-catalog.contains_content_items`